### ISSUE TITLE
usability questions

### ISSUE THREAD

**Issue by mihai-sysbio** [LCSB-BioCore/FBCModelTests.jl#61]
Thanks for putting together these test suites, very nice work!
I have some usability questions (possibly issues):

A) Minor: it's unclear what model formats are accepted, since it's just a string of the model file. The example points to json, which is not what I would have expected.

B) Minor: it's unclear to me why there is the need to have 3 different docker images instead of one. As a user, I'd prefer to have one image that includes all everything (including dependencies), and pass different commands to it with `docker run`.

C) Major: I can't seem to run the Memote-style test on [yeast-GEM](https://github.com/SysBioChalmers/yeast-GEM/tree/main/model):
```
FBCModelTests.Memote.generate_report("yeast-GEM.mat", Tulip.Optimizer)
```
since it ends up in an infinite loop that prints:
```
^C┌ Warning: Increase regularizations to 1.4901161193847656e-6
└ @ Tulip ~/.julia/packages/Tulip/SRdil/src/IPM/HSD/step.jl:47
```

**Comment by stelmo** [comment]
Ah, thanks for finding these issues! I will look into C this week and get back to you.

**Comment by stelmo** [comment]
Actually, I have put time aside for this at the end of next week

**Comment by exaexa** [comment]
Hi,
just a note re. B) with the docker images -- I wanted to separate off the FROG part that does only comparisons (tiny docker) from the one that also has to bundle the solvers and generate the reports. This is subject to change/simplify (I'll probably make 1 generic docker and have the other ones only as install/entrypoint patches over that).

The solver-in-docker problem is not a nice one though; I'd love to be able to somehow get gurobi licenses in there :]

**Comment by mihai-sysbio** [comment]
> The solver-in-docker problem is not a nice one though; I'd love to be able to somehow get gurobi licenses in there :]

How about mounting the license file at the expected location?

**Comment by exaexa** [comment]
yeah that should work, I really just need to find a moment try it (meaning I need to get the license etc. :D )

**Comment by mihai-sysbio** [comment]
This might come in handy, although it's valid only for a short window:

<img width="301" alt="image" src="https://user-images.githubusercontent.com/23480589/234227794-967dea3e-f138-42fa-8c3b-04e93b5ee901.png">

**Comment by mihai-sysbio** [comment]
> Next week we will release v.0.3.0
 Edit: we will soon release v0.3.0
 
 Is there any news regarding the upcoming release?

**Comment by exaexa** [comment]
Hi,
ok whoops, looks like someone (me) forgot to pull the trigger. Will release the 0.3 now :]
-mk

**Comment by exaexa** [comment]
(done)
